src/p2p/Lost.ts

```diff
@@ -271,7 +271,9 @@ export function init() {
             {}
           )
         } else {
-          res = await Comms.ask(targetNode, payload.route, payload.message)
+          error(`proxy handler route is not get_trie_hashes`)
```

Comment:
can we add what exact route was sent to log message ? without log injection vuln.

Reply:
will do